website/src/components/wordpress.tsx

```diff
@@ -125,9 +211,8 @@ const PullWords = (props: {
   const [{ data }] = Dailp.useDocSliceQuery({
     variables: {
       slug: props.slug,
-      // Convert our inclusive 1-indexed range into an exclusive 0-indexed range.
-      start: props.first - 1,
-      end: props.last ?? props.first,
+      start: props.first,
```

Comment:
Reason for change: In the backend, the `start` and `end` indices are ultimately 1-indexed, so I changed this because previously it was converting it to `0-index`.